test/vault/secrets/kvv2_test.clj

```diff
@@ -0,0 +1,133 @@
+(ns vault.secrets.kvv2-test
```

Comment:
In addition to the comment below about intent, I think my objections about testing http-client interaction from this test ns would go away more if there was a test of the secret engine itself, using the mock client. Pretty basic unit tests, but just asserting create, read, list, delete work as expected.

Reply:
Agreed, definitely going to test the mock client like you're suggesting, measuring known external state through client interactions (eg. write a secret, then read it back, etc.)